x/cdp/abci.go

```diff
@@ -20,6 +20,16 @@ func BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock, k keeper.Keeper)
 
 	params := k.GetParams(ctx)
 
+	// only run CDP Begin Blocker every `BeginBlockerExecutionBlockInterval` blocks
+	blockHeight := ctx.BlockHeight()
+
+	if blockHeight%params.BeginBlockerExecutionBlockInterval != 0 {
```

Comment:
On further thought here -- I believe we should only skip Synchronize and Liquidate @galxy25 @rhuairahrighairidh.  It's important to keep the pricefeed market status and global interest factor up to date for transactions, keeper liquidations, etc

Reply:
agreed, updated